lib/simctl.js

```diff
@@ -621,6 +621,17 @@ async function getDevices (forSdk, platform) {
  *                  platform version.
  */
 async function getRuntimeForPlatformVersion (platformVersion, platform = 'iOS') {
+  // Try it with --json flag
+  try {
+    const {stdout} = await simExec('list', 0, ['runtimes', '--json']);
+    for (const {version, identifier, name} of JSON.parse(stdout).runtimes) {
+      if (version === platformVersion && name.toLowerCase().startsWith(platform.toLowerCase())) {
+        return identifier;
```

Comment:
ah...
Should here return only `12-1-1` in `com.apple.CoreSimulator.SimRuntime.iOS-12-1-1`?
(travis failure)